### PR TITLE
Review details form

### PR DESCRIPTION
**Pull request by vlad0**
Create form for user data. 
Added validations.
Rated items are added to the form.

### CODE REVIEW

File: src/client/app/review/items-review/items-review.component.html [hunk 1]
```diff
@@ -2,40 +2,19 @@
...
-      <div class="col-md-6 col-sm-6 col-xs-6">
-        Шопска салата
+  <div class="panel-body" *ngIf="listItems.length > 0">
+    <div class="row" *ngFor="let item of listItems">
+      {{item.title}}
+      <button (click)="removeItem(item)" type="button" class="btn btn-danger btn-xs">
+        <span class="glyphicon glyphicon-remove" aria-hidden="true"></span>
+      </button>
+      <div>
+        <rating [(ngModel)]="item.rating"
```
Comment (Elinos) [inline src/client/app/review/items-review/items-review.component.html]: I'm sure I saw the same lines in the other PR, I don't know why we review PRs to feature branch before we actually merge the review-page

File: src/client/app/review/review-details/review-details.component.html [hunk 2]
```diff
@@ -1,12 +1,38 @@
...
-<email></email>
-<phone></phone>
-<comment></comment>
-<items-review></items-review>
-<div class="checkbox">
-  <label>
-    <input type="checkbox" value="">
-    I want to receive promotions
-  </label>
+<!--REMOVE!!!!!!!!!!!!!!!-->
```
Comment (Elinos) [inline src/client/app/review/review-details/review-details.component.html]: Do we need this?

File: src/client/app/review/review-details/review-details.component.ts [hunk 3]
```diff
@@ -1,8 +1,34 @@
...
+
+  public submit(formValue: any) {
+    console.log("Form values: ", formValue);
+    console.log("Items: ", this.items);
+  }
+
+  private buildForm() {
+    let emailRegex = VALIDATION_REGEX.email;
+
+    this.form = this.fb.group({
```
Comment (Elinos) [inline src/client/app/review/review-details/review-details.component.ts]: What is "fb"?

Comment (vlad0) [inline src/client/app/review/review-details/review-details.component.ts]: form builder

File: src/client/app/global.vars.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+let host = 'http://localhost:5000'
```
Comment (ivopashov) [inline src/client/app/global.vars.ts]: I made this file. Why is it in your PR?

Comment (vlad0) [inline src/client/app/global.vars.ts]: I needed it.

**Pull request merged**